docker/opbeans/python/entrypoint.sh

```diff
@@ -7,7 +7,11 @@ if [[ -f /local-install/setup.py ]]; then
     cd ~/local-install && python setup.py install
     cd -
 elif [[ $PYTHON_AGENT_VERSION ]]; then
+  if [[ "$PYTHON_AGENT_VERSION" != "latest" ]]; then
     pip install -q -U elastic-apm==$PYTHON_AGENT_VERSION
+  else
```

Comment:
this duplicates the default condition, can this just not be set if the latest is required?

Reply:
sure, let's remove the spaghetti code